### DISCUSSION TITLE
Rankup PlaceholderAPI Requirements

### DISCUSSION THREAD

**Discussion by BirminghamBoy** [Zrips/CMI#8119] [Feature Requests]
I recently stumbled across a rankup plugin that allows for PlaceholderAPI rankup requirements. In other words, the configurer can define a placeholder and what the output should be in order to fulfill the requirement (e.g., equals, greater than, less than, string, etc). This effectively opens up unlimited possibilities and make it possible to integrate virtually all of my plugins with the rankup feature.

[Click here](https://github.com/zorbeytorunoglu/thebestranks/wiki/How) for more information on how the aforementioned plugin works. 

I don't particularly want to switch as I have used CMI's rankup feature for years, but this is a feature I've been needing for some time.

**Comment by CloudeLecaw** [comment]
simliar to this might worth to add it here too

https://github.com/Zrips/CMI/discussions/7777    <- see stats needs to rank up
